tests/support_server/windows/win2019_boot.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Boot Windows server 2019 and wait samba_ad job child tests
+# Maintainer: mmartins <mmartins@suse.com>
+
+use strict;
+use warnings;
+use base 'windowsbasetest';
+use lockapi qw(mutex_create mutex_wait);
+use testapi;
+use mmapi;
+
+sub run {
+    my $self = shift;
+    # $self->wait_boot(bootloader_time => $timeout, nologin => $nologin);
+    assert_screen "windows-installed-ok", timeout => 400;
+    send_key "ctrl-alt-delete";
+    assert_screen "windows_server_login", timeout => 60;
+    type_string "N0tS3cr3t@";
+    send_key "ret";
+    #some times server_manager windows slow to open, fix waiting few seconds more...
+    assert_screen "wint_manage_server", timeout => 45;
```

Comment:
```suggestion
    assert_screen "wint_manage_server", timeout => 90;
```
The openQA can be very slow 😄

Reply:
Yeap, very slow. Suggestion accepted. :)